Shading of jdbc

In my project am getting 
```java
Caused by: java.lang.ArrayStoreException: org.apache.http.impl.cookie.BasicPathHandler
	at org.apache.http.impl.cookie.RFC6265CookieSpecProvider.create(RFC6265CookieSpecProvider.java:108) ~[httpclient-4.5.2.jar:4.5.2]
	at org.apache.http.client.protocol.RequestAddCookies.process(RequestAddCookies.java:152) ~[jdbc-1.1.2.jar:na]
	at org.apache.http.protocol.ImmutableHttpProcessor.process(ImmutableHttpProcessor.java:132) ~[httpcore-4.4.4.jar:4.4.4]
	at org.apache.http.impl.nio.client.MainClientExec.prepareRequest(MainClientExec.java:520) ~[httpasyncclient-4.1.1.jar:4.1.1]
	at org.apache.http.impl.nio.client.MainClientExec.prepare(MainClientExec.java:146) ~[httpasyncclient-4.1.1.jar:4.1.1]
	at org.apache.http.impl.nio.client.DefaultClientExchangeHandlerImpl.start(DefaultClientExchangeHandlerImpl.java:124) ~[httpasyncclient-4.1.1.jar:4.1.1]
	at org.apache.http.impl.nio.client.InternalHttpAsyncClient.execute(InternalHttpAsyncClient.java:141) ~[httpasyncclient-4.1.1.jar:4.1.1]
	at org.apache.http.impl.nio.client.CloseableHttpAsyncClient.execute(CloseableHttpAsyncClient.java:74) ~[httpasyncclient-4.1.1.jar:4.1.1]
	at org.apache.http.impl.nio.client.CloseableHttpAsyncClient.execute(CloseableHttpAsyncClient.java:107) ~[httpasyncclient-4.1.1.jar:4.1.1]
	at org.apache.http.impl.nio.client.CloseableHttpAsyncClient.execute(CloseableHttpAsyncClient.java:91) ~[httpasyncclient-4.1.1.jar:4.1.1]
```

and if we look closer it's a conflict of library's hence this
```java
    at org.apache.http.client.protocol.RequestAddCookies.process(RequestAddCookies.java:152) ~[jdbc-1.1.2.jar:na]
```

I think if we shade `jdbc` it will help

--- a/modules/jdbc/pom.xml
+++ b/modules/jdbc/pom.xml
@@ -40,10 +40,28 @@
                     </execution>
                 </executions>
                 <configuration>
+                    <transformers>
+                        <transformer
+                            implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer"/>
+                        <transformer
+                            implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
+                    </transformers>
                     <relocations>
                         <relocation>
                             <pattern>com.google.common</pattern>
                             <shadedPattern>org.testcontainers.shaded.com.google.common</shadedPattern>
+                        </relocation>
+                        <relocation>
+                            <pattern>org.glassfish</pattern>
+                            <shadedPattern>org.testcontainers.shaded.org.glassfish</shadedPattern>
+                        </relocation>
+                        <relocation>
+                            <pattern>javax.ws.rs</pattern>
+                            <shadedPattern>org.testcontainers.shaded.javax.ws.rs</shadedPattern>
+                        </relocation>
+                        <relocation>
+                            <pattern>com.fasterxml.jackson</pattern>
+                            <shadedPattern>org.testcontainers.shaded.com.fasterxml.jackson</shadedPattern>
                         </relocation>
                     </relocations>
                     <filters>
@@ -55,10 +73,22 @@
                                 <exclude>META-INF/*.RSA</exclude>
                             </excludes>
                         </filter>
+                        <filter>
+                            <artifact>*:*</artifact>
+                            <excludes>
+                                <exclude>META-INF/services/com.fasterxml.jackson.core.*</exclude>
+                                <exclude>META-INF/services/com.github.dockerjava.api.command.*</exclude>
+                                <exclude>META-INF/services/javax.ws.rs.ext.*</exclude>
+                                <exclude>META-INF/services/org.glassfish.hk2.extension.*</exclude>
+                                <exclude>META-INF/services/org.jvnet.hk2.external.generator.*</exclude>
+                                <exclude>META-INF/services/org.glassfish.jersey.internal.spi.*</exclude>
+                            </excludes>
+                        </filter>
                     </filters>
                     <artifactSet>
                         <excludes>
                             <exclude>com.google.guava:*</exclude>
+                            <exclude>org.glassfish.*:*</exclude>
                         </excludes>
                     </artifactSet>
                     <promoteTransitiveDependencies>true</promoteTransitiveDependencies>

